ui/hooks/gasFeeInput/useGasFeeInputs.js

```diff
@@ -61,7 +62,7 @@ export function useGasFeeInputs(
   defaultEstimateToUse = 'medium',
   transaction,
   minimumGasLimit = '0x5208',
-  editGasMode,
+  editGasMode = EDIT_GAS_MODES.MODIFY_IN_PLACE,
```

Comment:
What's the reasoning behind this change? This is probably okay but I'm just curious.

Reply:
`EDIT_GAS_MODES.MODIFY_IN_PLACE` is widely used gasMode, thus I added it to avoid passing each time.

Reply:
Okay. I'm wondering if we should watch out when setting default props in the future. The reason is that if for some reason this default gets changed, it may end up breaking tests and code that rely on that default. It may lead to less brittle code in the future if we are more explicit about this than less.

Reply:
I agree default should probably be the locally default value and not the most frequently used value. I will remove this in following PRs.